web/server/web_server.c

```diff
@@ -85,6 +110,15 @@ void web_client_update_acl_matches(struct web_client *w) {
 
     if(!web_allow_badges_from || simple_pattern_matches(web_allow_badges_from, w->client_ip))
         w->acl |= WEB_CLIENT_ACL_BADGE;
+
+    if(!web_allow_mgmt_from || simple_pattern_matches(web_allow_mgmt_from, w->client_ip))
+        w->acl |= WEB_CLIENT_ACL_MGMT;
+
+    if(!web_allow_streaming_from || simple_pattern_matches(web_allow_streaming_from, w->client_ip))
+        w->acl |= WEB_CLIENT_ACL_STREAMING;
+
+    if(!web_allow_netdataconf_from || simple_pattern_matches(web_allow_netdataconf_from, w->client_ip))
+        w->acl |= WEB_CLIENT_ACL_NETDATACONF;
 }
```

Comment:
It may be late at night, but why don't we check here if management is allowed or not, using both the IP ACL and the port ACL?

Reply:
I can't do it there, we only pass the client information. The port_acl comes from the server.

I don't see much difference with the check being done at https://github.com/netdata/netdata/blob/dd4241011e96f835052a078b292d3e9a8df69a2a/web/server/web_client.h#L122

Plus, I like the two separate parameters better, it's very clean this way (we could actually even have separate log entries on why the request was denied, if we wanted to. I didn't think of doing that, but it wouldn't be a bad thing to add).

Reply:
Made changes to set it at the end of that method.